docs/ota-client-guide/modules/ROOT/pages/supported-boards.adoc

```diff
@@ -39,12 +39,12 @@ Your images will also need network connectivity to be able to reach an actual OT
 
 == Minimum hardware requirements for controllers (ECUs)
 
-The aktualizr binary is a lightweight {cpp} application between 2-5 MB in size. It uses a minimum amount of resources when idle.
+The aktualizr binary is a lightweight {cpp} application between 2-5 MB in size. It uses around 20 MB RAM, although some of that comes from shared libraries. It uses a minimum amount of other resources when idle.
 
 The following hardware is required for your primary ECU:
 
-* At a minimum, aktualizr needs **16 MB of RAM** and **128 MB of storage** to run.
-* We recommend that you use a controller with **128 MB of RAM** and **512 MB storage** -- especially if you if your plan to process large, complex sofware updates.
+* At a minimum, aktualizr needs **64 MB of RAM** and **128 MB of storage** to run.
```

Comment:
Sorry that didn't notice earlier, but I think the phrasing here is a bit confusing. Should we better say "At a minimum, system running aktualizr..."?

Reply:
I agree.
```suggestion
* At a minimum, the board or device aktualizr is running on will need **64 MB of RAM** and **128 MB of storage**.
```